ui/shared/components/panel/family/Family.jsx

```diff
@@ -221,25 +223,38 @@ class Family extends React.PureComponent {
     })
   }
 
+  familyHeader = () => {
+    const { family, showFamilyPageLink } = this.props
+    const content = showFamilyPageLink ?
+      <Link to={`/project/${family.projectGuid}/family_page/${family.familyGuid}`}>{family.displayName}</Link> :
+      family.displayName
+    return <InlineHeader size="small" content={content} />
+  }
+
   render() {
     const {
       project, family, fields, rightContent, compact, useFullWidth, disablePedigreeZoom, disableEdit,
-      showFamilyPageLink, annotation, hidePedigree, toggleDetails,
+      annotation, hidePedigree, toggleDetails, updateFamily: dispatchUpdateFamily,
     } = this.props
 
     if (!family) {
       return <div>Family Not Found</div>
     }
 
+    const isEditable = !disableEdit && project.canEdit
+
     let leftContent = null
     if (!hidePedigree) {
       const familyHeader = (
-        <InlineHeader
-          key="name"
-          size="small"
-          content={showFamilyPageLink ?
-            <Link to={`/project/${family.projectGuid}/family_page/${family.familyGuid}`}>{family.displayName}</Link> :
-            family.displayName}
+        <BaseFieldView
+          field="familyId"
+          idField="familyGuid"
```

Comment:
does `idField` need to be set here? it's not required and I don't think it's used by the `familyHeader`

Reply:
Its used in `BaseFieldView` to generate a unique ID for the modal thats opened to do the editing. Its not required because if its not editable theres no need, but since this can be editable we should provide it